Modules/Filtering/AnisotropicSmoothing/itk-module.cmake

```diff
@@ -3,7 +3,7 @@ anisotropic smoothing. This is an image denoising technique that strives for
 preserving edges on the images while smoothing regions of uniform intensity.
 This type of filtering is convenient as a preprocessing stage of segmentation
 algorithms. You may find useful as well the filters in the
-\\\\ref ITKCurvatureFlow module and the \\\\ref TKSmoothingModule.")
+\\\\ref ITKCurvatureFlow module and the \\\\ref ITKSmoothing.")
```

Comment:
Perhaps add the word "module"here, as in `the \\\\ref ITKSmoothing module` ?

Reply:
ITK's scripts automatically add a `Module` prefix to these, so no need for that. Please have a look at the Doxygen files. In fact, I removed the redundant `module` wording in here in 5cb6105.